packages/sanes-chrome-extension/src/routes/account/index.dom.spec.ts

```diff
@@ -112,7 +112,8 @@ withChainsDescribe('DOM > Feature > Account Status', () => {
 
     // Check for the link
     const links = TestUtils.scryRenderedDOMComponentsWithTag(accountStatusDom, 'a');
-    expect(links.length).toBe(1);
+    //Amount of links calculation: 1 link for the blockExplorerUrl + 1 link for T&C
+    expect(links.length).toBe(2);
```

Comment:
Can't we change the test in a way to better search the link we look for? This test is about

> generates a link inside transaction box for an ethereum transaction

so we should search in the transaction box only and ignore other parts of the screen. Checking the T&C could be a different test that needs no transaction logic.

Reply:
Not required any more because "terms" has been moved to drawer menu